src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_driver_distraction_notification.cc

```diff
@@ -125,15 +125,29 @@ void OnDriverDistractionNotification::Run() {
   (*on_driver_distraction)[strings::msg_params][mobile_notification::state] =
       state;
 
+  const auto& policy_handler = application_manager_.GetPolicyHandler();
+
   const auto lock_screen_dismissal =
-      application_manager_.GetPolicyHandler().LockScreenDismissalEnabledState();
+      policy_handler.LockScreenDismissalEnabledState();
 
   if (lock_screen_dismissal &&
       hmi_apis::Common_DriverDistractionState::DD_ON == state) {
     (*on_driver_distraction)
         [strings::msg_params]
         [mobile_notification::lock_screen_dismissal_enabled] =
             *lock_screen_dismissal;
+
+    if (*lock_screen_dismissal) {
+      const bool result = AddLockScreenDismissalWarningToMessage(
```

Comment:
@IGapchuk you can use app_mngr , why you creating new pricy method?

Reply:
@LuxoftAKutsan Fixed in 048fe1415fce72016340c817c56367238ee052f6